pkg/authorization/authorizer/scope/converter.go

```diff
@@ -122,6 +123,15 @@ func (userEvaluator) ResolveRules(scope, namespace string, clusterPolicyGetter r
 	}
 }
 
+// escalatingScopeResources are resources that are considered escalating for scope evaluation
+var escalatingScopeResources = []unversioned.GroupResource{
+	{Group: kapi.GroupName, Resource: "secrets"},
+	/*imageapi.GroupName*/ {Group: "", Resource: "imagestreams/secrets"},
```

Comment:
> I don't really buy that, what do we gain from two fewer import statements?

Can be split out as a separate component.  It doesn't care whether various api groups are present.